SimpleTree blog -

hacke1978 opened the pull request:
cannot finish a paragraph with a picture so asd-line is added three times.

I do not find any more errors. HTML looks fine except this. I was not sure about one more:

It looks like the other entries, which are infact all code sprints and it is also tagged like this.

About content. I tried to stick just to some "fency" PCL related stuff in my tool and avoided the more statistical/forestry related contents. In majority I just use screenshots taken from to open access publications.

VictorLamoine commented on blogweb/index.rst:
```diff
@@ -40,6 +40,9 @@ the logos below to go to the respective sprint blog page.
   :target: http://www.pointclouds.org/blog/gsoc14
   :class: foo
+.. image:: _static/simple_tree.png
+  :target: http://www.pointclouds.org/blog/gsoc14
```
This is wrong, you want to point to `http://www.pointclouds.org/blogweb/simpletree`

VictorLamoine commented on blogweb/simpletree/hackenberg/index.rst:
```diff
@@ -0,0 +1,28 @@
+--------
+
+I am currently a PhD candidate at the former Chair of Forest Growth (University of Freiburg). My research interests include
+geometrical tree modelling methods - in the field of computational forestry often reffered to as Quantitative Structure Modelling.
```
There is no need for a carriage return here.

VictorLamoine commented on blogweb/simpletree/conf.py:
```diff
@@ -0,0 +1,226 @@
+# -*- coding: utf-8 -*-
+#
+# HRCS Website documentation build configuration file, created by
```
You might want to change HRCS to SimpleTree

VictorLamoine commented on blogweb/simpletree/conf.py:
```diff
@@ -0,0 +1,226 @@
+master_doc = 'index'
+
+# General information about the project.
+project = u'HRCS Website'
```
Also here

VictorLamoine commented on blogweb/simpletree/conf.py:
```diff
@@ -0,0 +1,226 @@
+#html_file_suffix = None
+
+# Output file base name for HTML help builder.
+htmlhelp_basename = 'HRCSWebsitedoc'
```
And here

VictorLamoine commented on blogweb/simpletree/conf.py:
```diff
@@ -0,0 +1,226 @@
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title, author, documentclass [howto/manual]).
+latex_documents = [
+  ('index', 'HRCSWebsite.tex', u'HRCS Website Documentation',
```
And here

VictorLamoine commented on blogweb/simpletree/conf.py:
```diff
@@ -0,0 +1,226 @@
+# One entry per manual page. List of tuples
+# (source start file, name, description, authors, manual section).
+man_pages = [
+    ('index', 'hrcswebsite', u'HRCS Website Documentation',
```
And here :)

VictorLamoine commented on blogweb/simpletree/index.rst:
```diff
@@ -0,0 +1,45 @@
+SimpleTree Code Sprint
```
It's not a code sprint so I would change the name here; maybe SimpleTree project fits better?

VictorLamoine commented:
Hello Jan, thank you for taking the time to write a blog article as requested!

I've added some remarks in the files themselves, here are others:
-  `simple_tree.png` and `simpletree.png` looks like duplicated images, remove one of them.
- The files ending with `~` are probably Gedit backup files, we don't need/want them in the repository, delete them. You can delete them with `find . -iname "*~" -exec rm -i {} \;` in a terminal. You can disable this Gedit behavior in Edit > Preferences > Editor tab > Create a backup of files before saving.
- Is `blogweb/simpletree/hackenberg/files/final_report.pdf` available anywhere else on the web? Maybe it's a better idea to point to your website rather than uploading it here?
- You don't use `blogweb/simpletree/images/pcl-hrcs.png`, delete it.

How to edit your pull request?
---
Just in case you are a git/github beginner, here are some instructions on how to edit your pull request:
```
cd blog
# Make your modifications
git add .
git commit -m "I have updated something"
git push
```

The pull request will be automatically updated with the changes.
I think this is really good advertising for the PCL and for your work :+1:

hacke1978 commented:
Hi Victor

I will take care of everything tomorrow.

Jan

On 02/12/15 18:05, Victor Lamoine wrote:
>
> Hello Jan, thank you for taking the time to write a blog article as 
> requested!
>
> I've added some remarks in the files themselves, here are others:
>
>   * |simple_tree.png| and |simpletree.png| looks like duplicated
>     images, remove one of them.
>   * The files ending with |~| are probably Gedit backup files, we
>     don't need/want them in the repository, delete them. You can
>     delete them with |find . -iname "*~" -exec rm -i {} \;| in a
>     terminal. You can disable this Gedit behavior in Edit >
>     Preferences > Editor tab > Create a backup of files before saving.
>   * Is |blogweb/simpletree/hackenberg/files/final_report.pdf|
>     available anywhere else on the web? Maybe it's a better idea to
>     point to your website rather than uploading it here?
>   * You don't use |blogweb/simpletree/images/pcl-hrcs.png|, delete it.
>
>
>     How to edit your pull request?
>
> Just in case you are a git/github beginner, here are some instructions 
> on how to edit your pull request:
>
> |cd blog
> # Make your modifications
> git add .
> git commit -m "I have updated something"
> git push
> |
>
> The pull request will be automatically updated with the changes.
> I think this is really good advertising for the PCL and for your work 
> :+1:
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/PointCloudLibrary/blog/pull/69#issuecomment-161366188>.
>

hacke1978 commented:
Thanks a lot Victor, you give great support.

I did the commit/push with the suggested improvements.

On 02/12/15 18:05, Victor Lamoine wrote:
>
> Hello Jan, thank you for taking the time to write a blog article as 
> requested!
>
> I've added some remarks in the files themselves, here are others:
>
>   * |simple_tree.png| and |simpletree.png| looks like duplicated
>     images, remove one of them.
>   * The files ending with |~| are probably Gedit backup files, we
>     don't need/want them in the repository, delete them. You can
>     delete them with |find . -iname "*~" -exec rm -i {} \;| in a
>     terminal. You can disable this Gedit behavior in Edit >
>     Preferences > Editor tab > Create a backup of files before saving.
>   * Is |blogweb/simpletree/hackenberg/files/final_report.pdf|
>     available anywhere else on the web? Maybe it's a better idea to
>     point to your website rather than uploading it here?
>   * You don't use |blogweb/simpletree/images/pcl-hrcs.png|, delete it.
>
>
>     How to edit your pull request?
>
> Just in case you are a git/github beginner, here are some instructions 
> on how to edit your pull request:
>
> |cd blog
> # Make your modifications
> git add .
> git commit -m "I have updated something"
> git push
> |
>
> The pull request will be automatically updated with the changes.
> I think this is really good advertising for the PCL and for your work 
> :+1:
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/PointCloudLibrary/blog/pull/69#issuecomment-161366188>.
>

VictorLamoine commented:
Looks good on my side.
@jspricke,what do you think?
